src/polyfills/abort-controller.js

```diff
@@ -24,6 +24,16 @@ class AbortController {
   /** */
   abort() {
     this.signal_.isAborted_ = true;
+    if (this.signal_.onabort_) {
+      const event = {
+        'type': 'abort',
+        'bubbles': false,
+        'cancelable': false,
+        'target': this.signal_,
+        'currentTarget': this.signal_,
+      };
+      this.signal_.onabort_(event);
```

Comment:
Oh, we need to ensure this doesn't double fire:

```js
controller.abort();
controller.abort();
```

Reply:
Good catch. Done.